book/reference/glossary.md

```diff
@@ -61,12 +61,19 @@
 
 ## Topical (Use Case Specific)
 ```{glossary}
-[basal slip](https://jupyterlab.readthedocs.io)
-  JupyterLab is the next-generation web-based user interface for Project Jupyter
-  intended to replace the JupyterNotebook interface.
-
-[subglacial water](https://jupyterlab.readthedocs.io)
-  JupyterLab is the next-generation web-based user interface for Project Jupyter
-  intended to replace the JupyterNotebook interface.
+[basal slip](https://en.wikipedia.org/wiki/Basal_sliding)
+  Basal slip or sliding is the act of a glacier sliding over the bed due to meltwater 
+  under the ice acting as a lubricant. This movement very much depends on the 
```

Comment:
Basal _slip_ isn't just from sliding over water, it also includes movement from (rock) basal deformation.

Reply:
Great point! Feel free to amend.